linode/manager
# change: [M3-8450] - Optimize event query timeframe: Reduce historical data fetch from 90 days to 7 days

## Description 📝
This change optimizes Cloud Manager's event querying by reducing the historical data range. Specifically, it limits the event query timeframe from the current 90 days to a more focused 7-day period. This adjustment aims to improve query performance and reduce unnecessary data retrieval while still providing relevant recent event information.

## Changes  🔄
- Previously we fetched the first 100 events but returned almost 500 results/records. These events pull from the last 90 days. Now we're returning events from the last 7 days up to a max of 25 results per query.

## Target release date 🗓️
8/19

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-08-15 at 1 00 23 PM](https://github.com/user-attachments/assets/c71889d7-44cf-4cdd-b154-7583a9623c30) | ![created-filter](https://github.com/user-attachments/assets/07961e19-0a5d-45a3-9133-e162db1a6330) |

## How to test 🧪
>[!note]
We are working to have this fully covered via E2E tests in an upcoming PR cc: @jdamore-linode 

### Prerequisites
- Checkout branch and observe you network tab

### Verification steps
- Go to http://localhost:3000/events
- Observe that initial load returns at most 25 events
- As you scroll observe subsequent queries fetch next 25
- Ensure there aren't any redundant events
- Ensure you're able to reach the end of your events list (90 days worth of events)

## As an Author I have considered 🤔

*Check all that apply*
